AlphaWallet/Core/Coordinators/WalletBalance/WalletBalanceFetcher.swift

```diff
@@ -138,7 +136,7 @@ class WalletBalanceFetcher: NSObject, WalletBalanceFetcherType {
             .initialOrNewTokensPublisher(forServers: [server])
             .receive(on: queue)
             .sink { [weak self] tokens in
-                guard let balanceFetcher = self?.balanceFetchers[safe: server] else { return }
+                guard let balanceFetcher = self?.balanceFetchers[server] else { return }
```

Comment:
The "safe" in `subscript(safe:)` function is not about array bounds checking so it returns an optional, not thread-safety. Are you intentionally removing it?

Reply:
subscript(safe:) it comes from ServersDictionary, to return optional value instead force unwrap, for now it threadSafeDictionary, that returns optional by default.

Reply:
Oh I see. Didn’t notice the type had changed. Great